Uruguay/URY_transform_load_dlt.py

```diff
@@ -290,16 +354,45 @@ def boost_silver():
             )
             .when(
                 (
-                    (
-                        col("econ2_lower")
-                        == "01 transferencias corrientes al sector publico"
-                    )
-                    | (
-                        col("econ2_lower")
-                        == "51 transferencias corrientes al sector publico"
+                    (col("year") < 2019)
+                    & (
+                        (
+                            (
+                                col("econ2_lower")
+                                == "01 transferencias corrientes al sector publico"
+                            )
+                            | (
+                                col("econ2_lower")
+                                == "51 transferencias corrientes al sector publico"
+                            )
+                        )
+                        & (
+                            (~col("func1").startswith("11 "))
+                            & (col("exp_type") != "Inversion")
```

Comment:
I thought this last condition shouldn't be necessary as CapEx definition above would have captured everything with exp_type = Inversion?

Reply:
@weilu 
You mean this condition?
https://github.com/dime-worldbank/mega-boost/blob/3fac948f709c742b5cbde77e34263c2baf6ca394/Uruguay/URY_transform_load_dlt.py#L294-L307

Since there were addition AND condition on econ, I thought it would be safer to explicitly exclude the inversion.